### ISSUE TITLE
shell: With zero "Applications" in the menu, unclear how to add the first

### ISSUE THREAD

**Issue by mvollmer** [cockpit-project/cockpit#14146]
When you don't have any manifests installed that put entries into the "Applications" category of the shell navigation, there will be no "Applications" section at all in the menu, and the "Edit" link disappears with it:

![image](https://user-images.githubusercontent.com/3228183/82784710-c4e02500-9e69-11ea-9ec2-b2340bb44c12.png)

There still is the "Applications" entry in the "Tools" section, but maybe we want to make it easier to discover how to add your first application.

**Comment by mvollmer** [comment]
(Sorry, I fell on the enter key in the middle of filling this out.)

**Comment by marusak** [comment]
@garrett I discusses this with you in the original PR and we decided to hide empty sections. WDYT about this case?

**Comment by garrett** [comment]
Like I said before, to install applications, you click on "Applications", just as before this PR landed. You can also search for "app" and it finds the "Applications" page. (This even works in different languages, as searches have a fallback to English, so "app" would work still.)

The Edit link is merely a shortcut link to the Applications page; it doesn't replace it.

If we had an empty Applications group only for the "edit" link, 90% of our users would have an empty, useless group, with no way to hide it.

Basically, this is 100% implemented as intended.

**Comment by garrett** [comment]
(Please reopen if this doesn't solve this or if you have a different idea how to handle it.)
